setup.py

```diff
@@ -16,7 +16,7 @@
 install_requires = [
     'six',
     # >= 8 9s because of breaking API change
-    'html5lib>=0.99999999',
+    'html5lib>=0.99999999,>=1.0b9',
```

Comment:
That might be the best way. The funny thing is that they managed to choose a pre-release version scheme for the saner versions, while the all-nines versions get interpreted as proper release versions, and so don't get selected normally.